micromanager_gui/multid_widget.py

```diff
@@ -15,6 +17,16 @@
 ICONS = Path(__file__).parent / "icons"
 
 
+@dataclass
+class SequenceMeta:
+    mode: Literal["mda"] | Literal["explorer"] = ""
```

Comment:
hey @fdrgsp, can you comment on why you wanted the default to be the empty string here?
This type annotation is no longer accurate (and `test_main_window` is now failing as a result)...  The type `Literal["mda"] | Literal["explorer"]` means "this value should either be the string `"mda"` or the string `"explorer"` ... but then we set it to neither of those.  thoughts?

Reply:
Hey @tlambert03, I've seen the tests failed and I was working on it...but basically if `"mode"` is set to `"mda"` as default, when using the explorer both `_on_explorer_frame ` and `_on_mda_frame` are executed.

Reply:
ah, k!